osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/infra/kafka/out/MeterReadingSerializer.java

```diff
@@ -0,0 +1,35 @@
+package org.opensmartgridplatform.adapter.kafka.da.infra.kafka.out;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+import org.apache.kafka.common.serialization.Serializer;
+import org.opensmartgridplatform.adapter.kafka.MeterReading;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// TODO - Rename and update for correct Avro message format.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)